WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -86,10 +89,15 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
 				foreach (var finishedCoinJoin in finishedCoinJoins)
 				{
-					var removed = trackedWallets.Remove(finishedCoinJoin.Wallet.WalletName);
+					var walletToRemove = finishedCoinJoin.Wallet;
+					var removed = trackedWallets.Remove(walletToRemove.WalletName);
 					if (!removed)
```

Comment:
Can also be:

```suggestion
					if (!trackedWallets.Remove(walletToRemove.WalletName))
```

Seems to be quite common. Feel free to ignore.

Reply:
True, I like this